pandas/core/dtypes/dtypes.py

```diff
@@ -772,13 +772,25 @@ def _creso(self) -> int:
     def unit(self) -> str_type:
         """
         The precision of the datetime data.
+
+        Examples
+        --------
+        >>> dtype = pd.DatetimeTZDtype(tz='dateutil/US/Central')
```

Comment:
Could you use `zoneinfo` for the `tz` argument for these examples?

Reply:
I changed them. 
I used the example from the reference for`DatetimeTZDtype` because I wasn't familiar with it. It also uses `dateutil`.  Here is the [link](https://pandas.pydata.org/docs/dev/reference/api/pandas.DatetimeTZDtype.html). 
Please tell me if I should change that.